addons/editor-theme3/theme3.js

```diff
@@ -144,6 +144,49 @@ const arrowShadowPath =
   "M12.71,2.44A2.41,2.41,0,0,1,12,4.16L8.08,8.08a2.45,2.45,0,0,1-3.45,0L0.72,4.16A2.42,2.42,0,0,1,0,2.44,2.48,2.48,0,0,1,.71.71C1,0.47,1.43,0,6.36,0S11.75,0.46,12,.71A2.44,2.44,0,0,1,12.71,2.44Z";
 const arrowShadowColor = "#231f20";
 
+const bevelFilters = [
```

Comment:
I suggest moving these filters to a separate SVG file because they're quite complex. They can then be accessed from the CSS like this:
```css
filter: url("%addon-self-dir%/filters.svg#bevelFilter");
```